frontend/utils/index.tsx

```diff
@@ -1,10 +1,25 @@
 interface LooseObject {
     id: number
 }
+
+interface LooseObject2 {
+    first_name: string
+}
+
 export function removeItemViaId<T extends LooseObject>(arr: Array<T>, value: number): Array<T> {
     return arr.filter((obj) => obj.id !== value)
 }
 
 export function isObjectEmpty(obj: Object): boolean {
     return obj === undefined ? true : Object.keys(obj).length === 0
 }
+
+export function compare(a: LooseObject2, b: LooseObject2) {
```

Comment:
Be descriptive on your function name. I cannot tell what you are trying to compare here.